lib/util/string.js

```diff
@@ -19,6 +19,14 @@
  * limitations under the License.
  */
 
-/**
- * @typedef {Request} KuzzleRequest
- */
+const Request = require('kuzzle-common-objects').Request;
+
+module.exports = {
+  /**
+   * @param {Request} request
+   * @returns {string}
+   */
+  getRequestRoute(request) {
```

Comment:
Hum.. I am not in favor of using some `util` namespaces when it can be avoided.

Further, IMO, this method is not really related to strings. 
Finally, I am wondering of the interest of having this level of detail anyway

1. In Kuzzle logs, we have the full stack trace and are  able to track where the error is generated from
2. From the end-user point of view, the error will be received in the callback of the request being sent.

In any case, we have some simple ways to retrieve which request generated the error.

My first option would simply be to stop repeating the controller/action in the error messages.
If this is really needed, we can:
* in controllers: hard-code the action
* in assertion utilities: move this piece of code into them.